pages/cart.html

```diff
@@ -0,0 +1,169 @@
+                <!---item 1 end-->
+    
+               <!---item 2 start--->                                
+            <div class="card_Container">
+                <div class="card_H">
+                    <div class="card_Img_H">
+                        <img class="card_Img_H" src="../assests/woman/dress5.jpg" alt="product_img"/>
+                    </div>
+                    <div class="card_Details">
+                        <h4>Blue Dress</h4>
+                           <small>by KM Fashion</small>
+                            <div class="card_Price">
+                                <span class="price_After">
+                                    Rs.500
+                                </span>
+                                <span class="price_Before">
+                                    Rs.999
+                                </span>
+                                <span class="discount">
+                                    (50% OFF)
+                                </span>
+                            </div>
+                                <button class="btn outline_A">
+                                    <span class="material-icons">delete_outline</span>
+                                        Delete
+                                </button> 
+                    </div>                    
+                </div>
+            </div>
+                <!--item 2 end here--->
+
+                </div>                                                                          
+          
+            <div class="checkout-container">
+                <div class="card-container">
+                    <div class="card_V">
+                        <div class="card_Details">
+                            
+                            <h2 >Price Details</h2><hr>
+                           
+                            <div class="card_Price">
+                                <p>Price(2 items)</p>
+                                <p>Rs. 1400</p>
+                            </div>
+                            <div class="card_Price">
+                                <p>Discount</p>
+                                <p>Rs. 1399</p>
+                            </div>
+                            <div class="card_Price">
+                                <p>Delivery Charges</p>
+                                <p>Rs. 0</p>
+                            </div>
+                            <hr>
+                            <div class="card_Price">
+                                <p>Total</p>
+                                <p>Rs. 1400</p>
+                            </div>
+                            <button class="btn btn_Success">
+                                Place Order
+                            </button>
+                        </div>
+                    </div>
+                </div>                                                                              
+            </div>
+        </div>
+    
+
+
+
+
```

Comment:
You can remove the extra spaces in the code

Reply:
removed space